src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorSlaveIntegrationTest.groovy

```diff
@@ -59,12 +58,35 @@ class IqPolicyEvaluatorSlaveIntegrationTest
     WireMock.configureFor("localhost", wireMockRule.port())
     givenThat(get(urlMatching('/rest/config/proprietary\\?.*'))
         .willReturn(okJson('{}')))
-    givenThat(put(urlMatching('/rest/ci/scan/.*'))
```

Comment:
I see a new test for `post` and `get`. I do not see a test for `put`. Was this removed inadvertently, or is it no longer needed?

Reply:
> I do not see a test for `put`

Yes, this was intentional.  With the polling change we have gone from uploading a scan (put) and using the resultant scan id to make another call (post) to do an evaluation to now uploading a scan for an evaluation (post) and using the resultant status id to poll (get) for the results.